src/docs/release-notes/R16/R16.mdx

```diff
@@ -0,0 +1,23 @@
+### Documentation Site
+
+Added search capability to doc site
+
+### New Components
+
+We have added new components to our [Angular](https://pxblue-components.github.io/angular) and [React Native](https://pxblue-components.github.io/react-native) component libraries:
+
+-   For Angular, we have added components for [Drawer](https://pxblue-components.github.io/angular-dev/?path=/info/components-drawer--readme), [ListItemTag](https://pxblue-components.github.io/angular-dev/?path=/info/components-list-item-tag--readme), [Spacer](https://pxblue-components.github.io/angular-dev/?path=/info/components-spacer--readme), and
+    [Scorecard](https://pxblue-components.github.io/angular-dev/?path=/info/components-score-card--readme), and [InfoListItem](https://pxblue-components.github.io/angular-dev/?path=/info/components-info-list-item--readme).
+-   For React Native, we have added a [Drawer](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--drawer) component.
+
+### React Native Reusable Workflow
+
+Innersourced a reusable [Authentication Workflow](https://github.com/pxblue/react-native-workflows) for React Native apps
+
+### Design Resources
+
+Published our stickersheet on Figma for public viewing
+
+### Miscellaneous
```

Comment:
Is this neccessary?

Reply:
I don't think it hurts anything to have it in there. Most of the time we are fixing bugs and stuff, so it might be nice for people to see that we are doing maintenance on existing materials. Some other libraries like to call out specific bugs that are fixed in each release...that might be a little overkill for us.

Reply:
Sounds good to me!  If people are ever asking about bug fix specifics I think this is something that we can integrate into github via tags instead of through our release notes.